docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md

```diff
@@ -0,0 +1,106 @@
+# SNS cycle management
+
+## The need to observe the canisters' cycles
+:::danger
+An SNS community must ensure that all SNS and dapp canisters
+have sufficient cycles by manually sending cycles when necessary to
+**each individual SNS canister**.
+:::
+
+1. _Manually_ means that someone has to monitor the cycles of the
+   canisters and actively make calls to send additional cycles to them
+   when the canisters' cycles balance get low.
```

Comment:
```suggestion
   when the canisters' cycles balance gets low.
```

Reply:
I think this is plural as it is the cycles balance of multiple canisters.